flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why???

Reply:
Because I don't have a aarch64 machine on hand and I cannot ensure that all used libraries are working well on aarch64.

Reply:
so nobody will test this either. that's not a good reason.

Reply:
Ok, I'll remove the line